__tests__/app.test.js

```diff
@@ -72,36 +72,83 @@ describe('GET /api/articles/:article_id', () => {
             .expect(200)
             .then(({ body }) => {
                 expect(body.article).toStrictEqual(

```

Comment:
Consider using a more flexible jest matcher here such as toMatchObject or objectContaining. If the table ever gets changed in the future we don't want to have to go back and edit our tests so we need to make sure this will still pass